README.md

```diff
@@ -1 +1,49 @@
-# Mobile-portfolio
\ No newline at end of file
+# Mobile design
+
+> Minimal mobile screen design inspired by Microverse, the design is a handoff on Figma and has been built using HTML & CSS.
+
+## Built With
+
+- HTML
+- CSS
+- Linters
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- You must have a code compiler installed on your machine such as (VS code or Atom).
+
+### Setup
+
+- Open you code editor.
+- Press cmd + shift + P and type gitclone and choose the first option.
+- Paste the repo link inside the field and press enter.
+
+## Authors
```

Comment:
- [OPTIONAL] I would recommend you to change `Authors` to `Author` as this is a solo project. Every solo project carries a single author.